in-band-manageability/Makefile

```diff
@@ -84,7 +84,28 @@ test: common-unit-test
 integration_test:
 	@echo "no integration tests to run, skipping"
 
-fuzztest: common-fuzztest
+fuzztest:
+	@echo "---MAKEFILE FUZZ TEST ALL---"
+	@# Help: Run ALL 18 fuzz tests - validation, parsing, and gRPC handlers
+	@go test ./cmd/inbd/ -fuzz=FuzzPackageValidation -parallel=1 -fuzztime=10s
```

Comment:
The duration of the fuzz tests should be configurable, in the same way done for the other agents. For example, when using the [common-fuzztest function](https://github.com/open-edge-platform/edge-node-agents/blob/main/common.mk#L45) the duration of each fuzz test is set using the `FUZZ_TEST` variable. Can you add the same here please?